docs/guides/upgrading.rst

```diff
@@ -22,6 +22,13 @@ Removed views
 
  * ``resources/file/download``
  * ``input/write_access``: mod/pages now uses the **access:collections:write** plugin hook.
+ * ``subscriptions/form/additions``: extend ``forms/notificationsettings/save`` instead
```

Comment:
Not the case anymore. Should use `subscriptions/settings/other` or whatever I named it an extension view.